### ISSUE TITLE
continues laser /tactical laser

### ISSUE THREAD

**Issue by yoel123** [yoel123/godot-yoel-space-sim-game#15]
can be created using a mesh and a raycast (to get the start and end point).

as for game/gameplay functionality, i don't want freespace like laser cannons, in my game the continues laser will be weak, used as point defense aginst rockets and bombs, it will be weak against shields so its not overpowered against fighters, another nerf will be that it will take a lot of energy so it will have a short use (like a 5-7 second timer). can be confused by flares as flares create both heat and aspect enough to confuse automated defenses.

basically its a protected energy weapon, unlike the other laser "bullets".
